File: cos/cos.js
// ATOMIC FUNCTION:cos
// TAGS: math，numeric
// LANG: node.js
// FUNCTION: Math.cos(val, precision)

var start_val=0;
var expected_end_val=1;

var end_val=Math.cos(start_val);

// TEST
if(end_val==expected_end_val){
    console.log('PASS');
}else{
    console.log('FAIL');
}

File: cos/cos.py

# ATOMIC_FUNCTION:  cos(val)
# TAGS: math，numeric
# LANG:  python
# FUNCTION: cos(val)
import math
start_val = 0
expected_end_val = 1

end_val = math.cos(start_val)


# TEST
if(end_val == expected_end_val):
    print("PASS")
else:
    print("FAIL")





